### PR TITLE
#148234831 Write Dummy Test

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### What does this PR do?

Write dummy test to make CI test pass.

#### Description of Task to be completed

- Write simple function to be tested
- Write simple test using function
- Add contribution guidelines to README

#### What are the relevant Pivotal Tracker stories?

[#148234831](https://www.pivotaltracker.com/story/show/148234831)

### CODE REVIEW

**Comment by njirap** [comment]
hey @BolajiOlajide Is the PR also a dummy PR?

**Comment by BolajiOlajide** [comment]
No, it isn't @njirap.

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,7 @@
 click==6.7
 Flask==0.12.2
-gunicorn==19.7.1
```
Comment (njirap) [inline requirements.txt]: @BolajiOlajide why is this removed?

Comment (BolajiOlajide) [inline requirements.txt]: No reason specifically. 
I was looking to clear the `requirements.txt` to install the main package and not it's dependencies since they come with the package when it's installed.

I'll add `gunicorn` to the `requirements.txt`

Comment (BolajiOlajide) [inline requirements.txt]: cc @njirap

File: tests/dummy.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"""Write a dummy function to multipy two numbers."""
+
+
+def multiplyDigits(num1, num2):
+    return num1 * num2
```
Comment (njirap) [inline tests/dummy.py]: we are not convinced we require this file

File: tests/test_dummy.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import unittest
```
Comment (njirap) [inline tests/test_dummy.py]: we are not convinced we require this file

Comment (BolajiOlajide) [inline tests/test_dummy.py]: This file exists so the Circle CI build will pass. Without it. there'll be no test to run and the CI build will fail.

Comment (njirap) [inline tests/test_dummy.py]: It's ok we can cherrypick your changes to the `README` and `.gitignore` and do away with the test files.

Comment (njirap) [inline tests/test_dummy.py]: Otherwise we can't add something for the purpose of deleting it later. Thanks for the PR!

Comment (BolajiOlajide) [inline tests/test_dummy.py]: Okay. I'll delete the test files.
